src/SourceBuild/content/eng/build.ps1

```diff
@@ -0,0 +1,78 @@
+[CmdletBinding(PositionalBinding=$false)]
+Param(
+  [string][Alias('c')]$configuration = "Release",
+  [switch] $clean,
+  [switch][Alias('bl')]$binaryLog,
+  [switch][Alias('nobl')]$excludeCIBinarylog,
+  [switch] $ci,
+  [switch] $prepareMachine,
+  [switch] $help,
+  [Parameter(ValueFromRemainingArguments=$true)][String[]]$properties
+)
+
+function Get-Usage() {
+  Write-Host "Common settings:"
+  Write-Host "  -configuration <value>  Build configuration: 'Debug' or 'Release' (short: -c). [Default: Release]"
+  Write-Host "  -binaryLog              Output binary log (short: -bl)"
+  Write-Host "  -help                   Print help and exit"
+  Write-Host ""
+
+  Write-Host "Actions:"
+  Write-Host "  -clean                  Clean the solution"
+  Write-Host ""
+
+  Write-Host "Advanced settings:"
+  Write-Host "  -ci                     Set when running on CI server"
+  Write-Host "  -excludeCIBinarylog     Don't output binary log (short: -nobl)"
+  Write-Host "  -prepareMachine         Prepare machine for CI run, clean up processes after build"
+  Write-Host ""
+}
+
+. $PSScriptRoot\common\tools.ps1
+
+# Set the NUGET_PACKAGES dir so that we don't accidentally pull some packages from the global location,
+# They should be pulled from the local feeds.
+$env:NUGET_PACKAGES="$PSScriptRoot\prereqs\packages\restored\"
```

Comment:
This will eventually need to get removed, so that when we don't pass -ci, the developer's machine can be used.

Reply:
I checked and I don't think we have any hardcoded expectations on where the package cache is today. I think moving this for both source build and non source build should be straightforward (but unrelated to this change).